modules/get-started/pages/intro-to-events.adoc

```diff
@@ -61,7 +61,7 @@ With the Bring Your Own Cloud (BYOC) offering, you deploy Redpanda in your own v
 
 == Redpanda Self-Managed versions
 
-You can deploy Redpanda in a self-hosted environment (Redpanda Self-Managed) or as a fully managed cloud service (Redpanda Cloud).
+You can deploy Redpanda in a self-managed environment (Redpanda Self-Managed) or as a fully managed cloud service (Redpanda Cloud).
 
```

Comment:
@Deflaimun Same comment here. Maybe we should leave this instance, since it'll read better & so we still have some reference to self-hosted in case users search "self-hosted"?

Reply:
reverted.